stable/application-mesh/templates/redirects.yaml

```diff
@@ -0,0 +1,79 @@
+{{- range $_, $redirect := .Values.redirects }}
+{{- $full_url := cat $redirect.source "" $redirect.source_uri -}}
+---
+apiVersion: networking.istio.io/v1beta1
+kind: VirtualService
+metadata:
+  name: {{ include "application-helpers.name" $ }}-{{ include "application-helpers.uniq-id" $full_url }}-redirects
```

Comment:
Our mesh is going to get messy if we create a lot of virtual services. It probably makes sense to create a single virtual service with a lot of rules.

Also the big open question is what happens if we have redirect for existing application domain? Same as gateway issue mentioned below